src/components/atoms/ButtonNG/ButtonNG.scss

```diff
@@ -12,7 +12,8 @@ $variants-enum: (
   ("primary", $white, $primary--live, $primary--live-hover),
   ("secondary", $white, opaque-white(0.2), opaque-white(0.25)),
   ("white", $primary--live, $white, $white),
-  ("dark", $white, opaque-black(0.4), opaque-black(0.6))
+  ("dark", $white, opaque-black(0.4), opaque-black(0.6)),
+  ("danger", $white, $input--invalid, darken($input--invalid, 10%))
```

Comment:
Can these from `constants.scss` be used instead? 

```scss
$input--invalid: #dc3545;
$input--invalid-focus: #ff7384;
```

Unless there are updated values from Design...

Reply:
`$input--invalid` is already being used, the `darken` is just used for darker hover color, it's not per design, because the design doesn't have hover colour.

Reply:
`$input--invalid-focus` looks like it's by some earlier design, even if maybe not explicit for button.

If still going with modification of `$input--invalid` maybe lighten instead of darken, since the primary has lighter color on hover...

🤷 But doesn't really matter... Your judgement is as good as mine. Kind of unfortunate can't ping design here with @ due to missing group.

Reply:
Makes sense, i'll lighten it instead.